### ISSUE TITLE
time-resource is not working properly

### ISSUE THREAD

**Issue by kkitai** [concourse/time-resource#19]
Hi.

I'm using concourse bosh release [3.3.1](http://bosh.io/releases/github.com/concourse/concourse?version=3.3.1) and applied `default_check_interval` property to bosh in order to change the default 1m check interval.
```
  jobs:
  - name: atc
    release: concourse
    properties:
      ...
      default_check_interval: 12h
```

When I set pipeline using time-resource like a following, I expect to be run my-task by 1m. But actually it looks like to be trigged by 12h.
```
resources:
- name: every-1m
  type: time
  check_every: 1m
  source:
    interval: 1m
    location: Asia/Tokyo

jobs:
- name: my-job
  plan:
  - get: every-1m
    trigger: true
  - task: my-task
    config:
```

![Imgur](http://i.imgur.com/989kqx0.png)

**Comment by jtarchie** [comment]
@kkitai, the `default_check_interval` is a property that defines how often each resource checks for a new version. It is not directly related to the time resource `interval`.

`default_check_interval` can be used to make sure all resources check less often. For example, your configuration will check every 12 hours for new git commits, too.

Remove `default_check_interval` from your bosh deployment, redeploy, and everything will work as expected.

**Comment by Go-zen-chu** [comment]
Hi, I am working with @kkitai and I just want to share some info about this issue.
What we have done is we updated our concourse to 3.3.1 and then changed the default_check_interval setting to 12h as @kkitai mentioned.
After that, some pipelines’ resources, that deployed before we have changed default_check_interval, keep checking their resource in 12h even we set their check_every setting to 1m.

This behavior is fixed when we destroy-recreated the pipeline, though it didn't fix when pause-unpaused pipeline.
I admit this is not a big issue, but I just wanted to share what we have faced. Thanks.
